tests/dataset_optimization_shared.py

```diff
@@ -8,7 +9,43 @@
 logger = get_logger(__name__)
 
 
-def cleanup(test_dataset: OptimizationDataset):
+def get_unique_id():
+    return os.getenv("UNIQUE_ID", "").replace(".", "-").replace("/", "-")
+
+
+def cleanup_conflict_by_unique_id(unique_id: typing.Optional[str]):
+    if unique_id:
```

Comment:
rather than indent the whole function, why not "short-curcuit" if `unique_id is None`?